src/SqlBindingConfigProvider.cs

```diff
@@ -35,10 +33,9 @@ internal class SqlBindingConfigProvider : IExtensionConfigProvider
         /// <exception cref="ArgumentNullException">
         /// Thrown if either parameter is null
         /// </exception>
-        public SqlBindingConfigProvider(IConfiguration configuration, IHostIdProvider hostIdProvider, ILoggerFactory loggerFactory)
```

Comment:
Just confirming this change is intentional. Unfamiliar with IHostIdProvider

Reply:
Yes, that's just used to generate the function ID for the trigger Leases table.